_episodes/07-development-setup.md

```diff
@@ -124,8 +135,10 @@ If an ``esmvaltool`` environment is already created following the lesson
 we should choose another name for the new environment in this lesson by:
 
 ~~~bash
-conda env create -n a_new_name --file environment.yml
+mamba env create --name a_new_name --file environment.yml
 ~~~
+This will create a new conda environment and install ESMValTool into it with a 
```

Comment:
```suggestion
This will create a new conda environment and install ESMValTool (with all dependencies that are needed for development purposes) into it with a 
```